_pages/arifcatalyst.yaml

```diff
@@ -0,0 +1,6 @@
+Arif Khan, [26.03.17 02:52]
+—-
+ githubHandle: arifcatalyst
+ pageUrl: arifcatalyst.github.io/Core-Java
+ timestamp: 2017-03-29
+ —-

```

Comment:
@arifcatalyst there's some extra characters near the top of the file, can you try typing the template into the file using the text editor rather than uploading a file? this may help